tox.ini

```diff
@@ -1,23 +1,31 @@
 [tox]
 minversion = 2.4
-envlist = lint,types,py{36,37,38},docs
+envlist = lint,types,py{36,37,38},docs,integration
 
 [testenv]
 deps =
     pretend
-    pytest<6.1 # https://github.com/pytest-dev/pytest-rerunfailures/issues/128
+    pytest
     pytest-cov
     pytest-socket
-    pytest-rerunfailures
+passenv =
+    PYTEST_ADDOPTS
+commands =
+    pytest --ignore-glob '*integration*.py' {posargs:--cov-report term-missing --cov-report html tests}
```

Comment:
I tried to use `-k 'not ntegration'`, but that caused pytest's collection to fail, because it still imports `test_integration.py`.

If this is too exotic, I'm fine with leaving `tox.ini` alone and just using `-k` in the GH Action.